recipe/meta.yaml

```diff
@@ -0,0 +1,132 @@
+
+requirements:
+  host:
+    - python
+    - setuptools
+    - wheel
+    - pip
+  run:
+    - python
+    - numpy >=1.22.0
+    - pandas >=1.5.0,<2.1.0
+    - scipy >=1.5.0
+    - scikit-learn >=1.3.0
+    - scikit-optimize >=0.9.0
+    - pyzmq >=20.0.0
+    - colorama >=0.4.4
+    - cloudpickle >=1.5.0
+    - click >=8.0.0
+    - shap >=0.42.0
+    - statsmodels >=0.12.2
+    - texttable >=1.6.2
+    - woodwork >=0.22.0
+    - dask-core >=2022.2.0,!=2022.10.1
+    - distributed >=2022.2.0,!=2022.10.1
+    - featuretools >=1.16.0
+    - nlp_primitives >=2.9.0
+    - networkx >=2.6,<3.2
+    - plotly >=5.0.0
+    - python-kaleido >=0.1.0
+    - ipywidgets >=7.5
+    - xgboost >=1.7.0
+    - catboost >=1.1.1
+    - lightgbm >=4.0.0
+    - matplotlib-base >=3.3.3
+    - python-graphviz >=0.13  # [not win]
+    - seaborn >=0.11.1
+    - category_encoders >=2.2.2,<=2.5.1.post0
+    - imbalanced-learn >=0.11.0
+    - pmdarima >=1.8.5
+    - sktime >=0.21.0
+    - lime >=0.2.0.1
+    - vowpalwabbit >=8.11.0
+    - tomli >=2.0.1
+    - packaging >=23.0
+    - black >=22.3.0
+    - holidays >=0.13,<0.21
+    - jupyter
+
+# Skip some tests:
+# AssertionError: assert '$PREFIX/lib/python3.9/site-packages/evalml' == '$SRC_DIR/evalml'
+{% set tests_to_skip = "test_get_evalml_root" %}
+# Something else not able to get the local dirs count
+{% set tests_to_skip = tests_to_skip + " or test_all_test_dirs_included" %}
+
+# these tests are failing because of a problem with class evalml/pipelines/components/transformers/encoders/target_encoder.py:TargetEncoder
+# it's very difficult to document what the root cause is, no reference in the upstream repo issues either
+{% set tests_to_skip = tests_to_skip + " or test_search_parameters_held_automl or test_transformer_transform_output_type" %}
+{% set tests_to_skip = tests_to_skip + " or test_all_transformers_check_fit or test_all_transformers_check_fit_input_type or test_serialization" %}
+{% set tests_to_skip = tests_to_skip + " or test_transformer_fit_and_transform_respect_custom_indices or test_components_support_nullable_types" %}
+{% set tests_to_skip = tests_to_skip + " or test_null_values_in_dataframe or test_cols or test_transform or test_smoothing" %}
+{% set tests_to_skip = tests_to_skip + " or test_get_feature_names or test_target_encoder_woodwork_custom_overrides_returned_by_components" %}
+{% set tests_to_skip = tests_to_skip + " or test_fast_permutation_importance_matches_slow_output" %}
+
+# skipping because of rounding issue with pandas.DataFrame calculation
+{% set tests_to_skip = tests_to_skip + " or assert_almost_equal" %}  # [linux and x86_64]
+# skipping other hard to debug tests
+{% set tests_to_skip = tests_to_skip + " or test_lda_numeric" %}  # [linux and x86_64]
+# skipping test failing on prefect on osx-arm64
+{% set tests_to_skip = tests_to_skip + " or test_results_getter" %}  # [osx or (linux and x86_64)]
+
+
+test:
+  imports:
+    - evalml
+  requires:
+    - pip
+    # - pytest >=7.1.2
```

Comment:
could you please remove the commented-out lines?

Reply:
It's useful and serves as some kind of documentation for the next person that will look at this feedstock in the future.